core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -845,6 +848,19 @@ public static ExceptionVerbosity annotatedServiceExceptionVerbosity() {
         return ANNOTATED_SERVICE_EXCEPTION_VERBOSITY;
     }
 
+    /**
+     * Enables {@link DefaultAddressResolverGroup} that resolves using JDK's built-in domain name
+     * lookup mechanism.
+     * Note that this resolver performs a blocking name lookup from the caller thread.
```

Comment:
- this resolver -> JDK's built-in resolver
- thread. -> thread, and thus this flag should be enabled only when the default asynchronous resolver does not work as expected.